spec/services/semester_spec.rb

```diff
@@ -0,0 +1,78 @@
+describe Semester do
+  it "Checking autumn semester calculated current in semester's January" do
+    Timecop.freeze '2020-01-01' do
+      expect(Semester.new("2019202001").current?).to be true
+    end
+  end
+  it "Checking autumn semester calculated current in semester's start (September 1st)" do
+    Timecop.freeze '2019-09-01' do
+      expect(Semester.new("2019202001").current?).to be true
```

Comment:
It think there is an extra zero in some of the date strings. Please take a look at it.
```suggestion
      expect(Semester.new("201920201").current?).to be true
```